desktop/src/gameover_scene_slc.nim

```diff
@@ -1,6 +1,15 @@
 import game_types
 import scene_management
 from game_input import wasClicked
+from text_renderer import renderTextCached
+
+proc drawGameOverText(game: Game) =
```

Comment:
we don't need a separate procedure for this, just move the code into the render proc